css/bulma.css

```diff
@@ -10032,12 +10005,12 @@ label.panel-block:hover {
 }
 
 .hero.is-white.is-bold {
-  background-image: linear-gradient(141deg, #e6e6e6 0%, white 71%, white 100%);
+  background-image: linear-gradient(141deg, #e8e3e4 0%, white 71%, white 100%);
```

Comment:
This is due to the adjust hue being executed at the same time than the adjust lightness, while before the adjust lightness was executed after the adjust hue (which does noting on white)

I consider this a bug fix